.env.sample

```diff
@@ -0,0 +1 @@
+ACTIVITY_DIGEST_RECIPIENT_EMAILS=group1@example.com
```

Comment:
I think `.env.example` is better, or vi rename `local.py.example`. Just so we are consistent.

Reply:
I'll have it in the local.py.example, and if you like the .env.example approach, we can move it at once.